xcode/Subconscious/Shared/Components/Common/MarkupTextViewRepresentable.swift

```diff
@@ -141,37 +159,6 @@ struct MarkupTextViewRepresentable: UIViewRepresentable {
             self.representable = representable
         }
 
-        /// NSTextStorageDelegate method
-        /// Handle markup rendering, just before processEditing is fired.
-        /// It is important that we render markup in `willProcessEditing`
-        /// because it happens BEFORE font substitution. Rendering before font
-        /// substitution gives the OS a chance to replace fonts for things like
-        /// Emoji or Unicode characters when your font does not support them.
-        /// See:
-        /// https://github.com/gordonbrander/subconscious/wiki/nstextstorage-font-substitution-and-missing-text
-        ///
-        /// 2022-03-17 Gordon Brander
```

Comment:
I _believe_ we no longer need to work via `NSTextStorage` at all anymore. We can render markup ephemerally without altering the underlying storage.

This comment had me worried that we'd lose support for proper unicode / emoji rendering but it seems to be behaving, please advise of any other edgecases to check.

Reply:
This may be a TK1 specific issue. The best thing to do is playtest on-device. Watch for issues with autocompletion, autocorrection, emoji, and emoji substitution.

Reply:
I tested this on-device and everything seemed well behaved to me.